src/mumble/SettingsKeys.h

```diff
@@ -89,6 +89,10 @@ const SettingsKey POSITIONAL_TRANSMIT_POSITION_KEY = { "transmit_position" };
 const SettingsKey JITTER_BUFFER_SIZE_KEY            = { "jitter_buffer_size" };
 const SettingsKey FRAMES_PER_PACKET_KEY             = { "frames_per_packet" };
 const SettingsKey RESTRICT_TO_TCP_KEY               = { "restrict_to_tcp" };
+const SettingsKey UDP_VOICE_INBOUND_ONLY_KEY        = { "udp_voice_inbound_only" };
+const SettingsKey UDP_VOICE_OUTBOUND_ONLY_KEY       = { "udp_voice_outbound_only" };
+const SettingsKey UDP_VOICE_BIDIRECTIONAL_KEY       = { "udp_voice_bidirectional" };
+const SettingsKey UDP_VOICE_AUTO_KEY                = { "udp_voice_auto" };
```

Comment:
Also here: I don't think "voice" is necessary as part of the names

Reply:
Done.